apps/api/src/api/services/phases/handlers/brla-teleport-handler.ts

```diff
@@ -22,87 +19,45 @@ import { StateMetadata } from "../meta-state-types";
 const PAYMENT_TIMEOUT_MS = 30 * 60 * 1000; // 30 minutes
 const EVM_BALANCE_CHECK_TIMEOUT_MS = 5 * 60 * 1000; // 5 minutes
 
+// Phase description: wait for the tokens to arrive at the Moonbeam ephemeral address.
+// If the timeout is reached, we assume the user has NOT made the payment and we cancel the ramp.
 export class BrlaTeleportPhaseHandler extends BasePhaseHandler {
   public getPhaseName(): RampPhase {
     return "brlaTeleport";
```

Comment:
Shall we rename the phase to something more accurate? 'brlaWaitForPayin' or similar?